Formula/gpgme.rb

```diff
@@ -12,7 +13,7 @@ class Gpgme < Formula
     sha256 "4a4c3e90175b02ded974a8a385b6d9799bcc2873c5a1087c558f350eb103934d" => :yosemite
   end
 
-  depends_on "gnupg2"
+  depends_on "gnupg"
```

Comment:
Colour me unsurprised 😕. Probably want to rewrite the `inreplace` and the `test`, I guess.